src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/98.diff.sql

```diff
@@ -0,0 +1,11 @@
+IF NOT EXISTS (
+    SELECT 1 
```

Comment:
Jist a nitpick... SELECT 1 has no effect of efficiency of EXISTS in SQL server, because SQL ignores column list in the SELECT clause specifically for (NOT)EXISTS statements.